tsconfig.json

```diff
@@ -12,7 +28,7 @@
     "allowJs": true,
     "baseUrl": ".",
     "types": ["jest"],
-    "lib": ["ESNext"]
+    "lib": ["ESNext", "DOM"]
```

Comment:
🤔

Reply:
for console

Reply:
Not sure I follow - why do we need Dom types available?

Reply:
Sorry, to be more specific TS was complaining about use of console and suggested adding the dom type. Node type would probably also work.

Reply:
where was it complaining?

Reply:
in a test - i took the console out. I guess we dont actually need it, just needed it at the time for debugging